### ISSUE TITLE
Common::Base subclasses for app/models

### ISSUE THREAD

**Issue by stevenjcumming** [department-of-veterans-affairs/va.gov-team#92439]

**Comment by AshleyGuerrant** [comment]
**Sprint reporting note:** this ticket was injected into Sprint 26. The dev started working on [another ticket in the epic ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92440)yesterday, and realized some work with the subclasses was not necessary - which means it will take significantly less time to complete, so they will have additional bandwidth.

**Comment by stevenjcumming** [comment]
Some models are blocked by an update required for Collections... They will be moved into a separate ticket(s). They include the following models:

- Messaging related models (including Folder)
- Prescription related models 
- AllTriageTeams
- TriageTeam
- EligibleDataCalls
